react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const NavBarHeaderLink = styled.a.attrs({
+  className: 'navBarHeaderLink',
+})`
+  -webkit-font-smoothing: antialiased;
+  color: white;
+  font-size: 1.4em;
+  font-style: bold;
+  line-height: 1;
+  padding: 5px;
+`;
+
+export const NavBarLi = styled.li.attrs({
+  className: 'navBarLi',
+})`
+  margin: 5px 0;
+  @media screen and (min-width: 800px) {
+    margin: 0;
+  }
+`;
+
+export const NavBarLink = styled(Link).attrs({
+  className: 'navBarLink',
+})`
+  color: #fff;
+  display: flex;
+  font-family: ‘BC Sans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
```

Comment:
Not sure why, by the `font-family` is not being picked up on the navbar links. I'd think this line would do it, but something else is negating it?

![navbar-font](https://user-images.githubusercontent.com/16946297/94949627-15116880-0496-11eb-8ee8-ef78838be756.png)